recipes/simage/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conan import ConanFile
+from conan.tools.files import get, rm, rmdir, collect_libs, copy
+from conan.tools.cmake import cmake_layout, CMake, CMakeDeps, CMakeToolchain
+import pathlib
+
+def get_include_root(conanfile: ConanFile, depname: str) -> str:
+    pth = pathlib.Path(conanfile.dependencies[depname].cpp_info.includedirs[0]).parent
+    return str(pth)
+
+
+class SImageConan(ConanFile):
+    name = "simage"
+    settings = "os", "arch", "compiler", "build_type"
+    description = "library for loading, manipulating and saving images, movies and audio"
+    license = "ISC"
+    homepage = "https://www.coin3d.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "library"
+    topics = ("image", "video")
+    options = {"shared" : [True, False],
+               "with_jasper" : [True, False],
+               "with_qt" : [5, 6, None],
+               "fPIC" : [True, False]}
+
+    default_options = {"shared" : False,
+                       "with_jasper" : False,
+                       "with_qt" : None,
+                       "fPIC" : True}
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def requirements(self):
+        self.requires("libsndfile/1.2.2")
+        self.requires("zlib/1.2.13")
+        self.requires("giflib/5.2.1")
+        self.requires("libjpeg/9e")
+        self.requires("libpng/1.6.40")
+        self.requires("libtiff/4.6.0")
+        self.requires("ogg/1.3.5")
+        self.requires("vorbis/1.3.7")
+        if self.options.with_jasper:
+            self.requires("jasper/4.0.0")
+        if self.options.get_safe("with_qt", 5):
+            self.requires("qt/5.15.10")
+        elif self.options.get_safe("with_qt", 6):
+            self.requires("qt/6.5.2")
```

Comment:
```suggestion
        if self.options.with_qt == 5:
            self.requires("qt/5.15.10")
        elif self.options.with_qt == 6:
            self.requires("qt/6.5.2")
```
The current code does not look correct to me.

Reply:
ah yes well spotted this was a silly error!